src/main/webapp/scripts/tests/comment-post-test.js

```diff
@@ -0,0 +1,69 @@
+import { buildRequestBody, makeApiCall } from '/scripts/comments.js';
+
+const VIDEO_ID = 'sample video id';
+const COMMENT_TEXT = 'sample comment';
+
+QUnit.module('Post a comment', {
+    before: loadClientAsync,
+    beforeEach: () => { sinon.stub(gapi.client, 'request'); },
+    afterEach: () => { gapi.client.request.restore(); },
+});
+
+QUnit.test('BuildRequestBody', function (assert) {
```

Comment:
You should be testing behavior of the module and not helper methods. There is a ToTT for this. You should be testing that given a video id and comment text to makeApiCall, it make the API call with the correct request body, its mocked response is converted to the expected returned value of the method.

Reply:
yap done! (I miss how java uses private functions to enforce that practice)